db/app.js

```diff
@@ -27,4 +29,10 @@ app.use((err, req, res, next) => {
     }
   });
 
+  app.use((err, req, res, next) => {
+    if (err) {
+      console.log(err);
```

Comment:
this error handler will just hand in present state if you get some kind of server error, fyi